ci-operator/step-registry/openshift/microshift/infra/rpm-install-aws/openshift-microshift-infra-rpm-install-aws-commands.sh

```diff
@@ -6,6 +6,7 @@ curl https://raw.githubusercontent.com/openshift/release/master/ci-operator/step
 source /tmp/ci-functions.sh
 ci_script_prologue
 trap_subprocesses_on_term
+trap_install_status_exit_code $EXIT_CODE_RPM_INSTALL_FAILURE
```

Comment:
This overrides the subprocess TERM trap, so we need something that runs both, probably?